### PR TITLE
docs: fix small CSS issues

### PR DESCRIPTION
**Pull request by andrewrynhard**
Adds a dashed line between struct field sections in generate configuration
markdown. Also improves the padding of the headers in the right sidebar.

Signed-off-by: Andrew Rynhard <andrew@rynhard.io>

### CODE REVIEW

File: pkg/machinery/config/encoder/markdown.go [hunk 1]
```diff
@@ -49,13 +49,15 @@ Appears in:
 
 {{ if $struct.Fields -}}
 
+<hr class="my-4 border-t border-dashed border-ui-border" />
+
 {{ range $field := $struct.Fields -}}
-<div class="dd">
+<div class="mx-1 font-bold">
```
Comment (andrewrynhard) [inline pkg/machinery/config/encoder/markdown.go:55]: Replaces the CSS in `main.css` with the "Tailwind way".

File: pkg/machinery/config/encoder/markdown.go [hunk 2]
```diff
@@ -49,13 +49,15 @@ Appears in:
...
 
 <code>{{ $field.Name }}</code>  <i>{{ encodeType $field.Type }}</i>
 
 </div>
-<div class="dt">
+<div class="ml-8">
```
Comment (andrewrynhard) [inline pkg/machinery/config/encoder/markdown.go:60]: Replaces the CSS in `main.css` with the "Tailwind way".

File: pkg/machinery/config/encoder/markdown.go [hunk 3]
```diff
@@ -89,8 +93,6 @@ Valid values:
 - {{ $tick }}{{ $value }}{{ $tick }}
 {{ end -}}
 {{- end }}
-
----
```
Comment (andrewrynhard) [inline pkg/machinery/config/encoder/markdown.go:93]: New `hr` element makes this redundant.

File: website/src/assets/css/main.css [hunk 4]
```diff
@@ -1,14 +1,3 @@
...
-div.dt {
-    margin-left: 2em;
-}
-
-div.dd {
-    padding-top: 0.5em;
-    padding-bottom: 0.5em;
-
-    font-weight: bolder;
-}
```
Comment (andrewrynhard) [inline website/src/assets/css/main.css:14]: Replaced with Tailwind equivalent.

File: website/src/components/OnThisPage.vue [hunk 5]
```diff
@@ -4,15 +4,14 @@
     <div class="h-screen overflow-y-auto">
       <ul>
         <li
-          v-for="(heading, index) in headings"
+          v-for="heading in headings"
           :key="`${page.path}${heading.anchor}`"
           :class="{
-            'border-t border-dashed border-ui-border':
-              index > 0 && heading.depth === 2,
```
Comment (andrewrynhard) [inline website/src/components/OnThisPage.vue:11]: Removes dashed border between `h2` elements in the right sidebar.

File: website/src/templates/MarkdownPage.vue [hunk 6]
```diff
@@ -97,4 +97,16 @@ code[class*="language-"],
...
+  @apply my-4 border-t border-dashed border-ui-border;
+}
+
+.dt {
+  @apply ml-8;
+}
+
+.dd {
+  @apply mx-1 font-bold;
+}
```
Comment (andrewrynhard) [inline website/src/templates/MarkdownPage.vue:111]: Tailwind equivalent of what was removed from `main.css`.

File: website/src/components/OnThisPage.vue [hunk 7]
```diff
@@ -4,15 +4,14 @@
...
           :key="`${page.path}${heading.anchor}`"
           :class="{
-            'border-t border-dashed border-ui-border':
-              index > 0 && heading.depth === 2,
+            'mt-3': heading.depth === 2,
             'font-semibold': heading.depth === 2,
             [`depth-${heading.depth}`]: true,
           }"
-          class="pl-4 pb-2"
+          class="pl-4 pb-1"
```
Comment (andrewrynhard) [inline website/src/components/OnThisPage.vue:14]: Makes the different top-level groups more distinguishable from each other.

**Review by smira — APPROVED** [review]
:+1:

**Pull request merged**